src/subscription_manager/entcertlib.py

```diff
@@ -160,12 +151,20 @@ def perform(self):
                         obsolete_certs.append(cont_access_cert)
                 log.info('Deleting obsolete content access certificate')
                 self.delete(obsolete_certs)
```

Comment:
Actually, I think we don't need this any more since this change has included the SCA cert in the 'local' list so the obsolete SCA certs should always delete in 'self.delete(rogue_serials)' (line 129)